src/utils/getSymbolScale.ts

```diff
@@ -8,7 +8,14 @@ export const getSymbolScale = (table: Table): Scale<string, SymbolType> => {
   const symbolCol = table.getColumn(SYMBOL, 'string')
   const uniqueValues = Array.from(new Set(symbolCol))
 
-  const symbolTypes = ['circle', 'triangle', 'square'] as SymbolType[]
+  const symbolTypes = [
```

Comment:
I think
```suggestion
  const symbolTypes: SymbolType[] = [
```
would remove the need for the type assertion.